webapp/client/src/components/DisplayFailureIcon.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from "prop-types";
+import { useTranslation } from "react-i18next";
+import { ReactComponent as FailureIcon } from "../assets/icons/failure_icon.svg";
+
+export default function DisplayFailureIcon({ title, errorDetails }) {
```

Comment:
Can we find a better name for this?
Maybe: 
``FailureMessageForm`` ?

Reply:
What about FailureMessageBox?

Reply:
That was the name before with the template: display_failure_icon, I just used that name for the react component and it is in fact quite convenient. We have a display component with a failure icon, wdyt?

Reply:
FailureMessageBox it is! :)